app/src/main/java/de/tum/in/tumcampusapp/cards/NextLectureCard.java

```diff
@@ -162,28 +163,27 @@ protected Notification fillNotification(NotificationCompat.Builder notificationB
         return notificationBuilder.build();
     }
 
-    public void setLectures(Cursor cur) {
-        do {
+    public void setLectures(List<de.tum.in.tumcampusapp.models.tumo.CalendarItem> calendarItems) {
+        for (de.tum.in.tumcampusapp.models.tumo.CalendarItem calendarItem: calendarItems) {
```

Comment:
Is there a reason for this overly long type? `CalendarItem` should do?

Reply:
NextLectureCard has a static class with public fields to *conveniently* store some values that are used in the card. I would use CalendarItem entity, but some fields types are different (dates), but I will try to change them and see for back-compatibility